client/src/components/DataFiles/DataFilesModals/DataFilesCompressModal.js

```diff
@@ -0,0 +1,175 @@
+import React, { useMemo } from 'react';
+import { useSelector, useDispatch, shallowEqual } from 'react-redux';
+import {
+  Button,
+  Modal,
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+  Form,
+  Input,
+  FormGroup
+} from 'reactstrap';
+import { LoadingSpinner, FormField, Icon, InlineMessage } from '_common';
+import { useHistory, useLocation } from 'react-router-dom';
+import { Formik } from 'formik';
+import PropTypes from 'prop-types';
+import * as yup from 'yup';
+import './DataFilesCompressModal.module.scss';
+
+const DataFilesCompressForm = ({ singleFile, formRef }) => {
+  const initialValues = {
+    filenameDisplay: `${singleFile || ''}`,
+    filetype: '.zip'
+  };
+  const validationSchema = yup.object().shape({
+    filenameDisplay: yup.string().required('The filename is required')
```

Comment:
Let's add a `trim` to prevent blank submissions
```suggestion
    filenameDisplay: yup
      .string()
      .trim('The filename cannot include leading and trailing spaces')
      .strict(true)
      .required('The filename is required')
```